doc/rtd/reference/datasources/ovf.rst

```diff
@@ -3,9 +3,35 @@
 OVF
 ***
 
-The OVF datasource provides a datasource for reading data from an
+The OVF datasource provides a generic datasource for reading data from an
 `Open Virtualization Format`_ ISO transport.
 
+What platforms support OVF
+--------------------------
+
+OFV is an open standard which is supported by various virtualization
```

Comment:
Sorry I was a bit late to the party, but OFV s/b OVF :)

Reply:
Ups, I missed that one. The commit message also contained OFV.